hoot-core/src/main/cpp/hoot/core/cmd/ConvertCmd.cpp

```diff
@@ -65,6 +68,14 @@ class ConvertCmd : public BaseCommand
 
     LOG_VART(args.size());
     LOG_VART(args);
+
+    bool writeBoundsFile = false;
```

Comment:
Somewhere down the road maybe we ought to move this duplicated code into the base class or create a new base derived on `BaseCommand` maybe something like `BoundedCommand` or something like that.  Just an idea.